pkg/ingester/ingester.go

```diff
@@ -103,6 +105,8 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.DurationVar(&cfg.MetadataRetainPeriod, "ingester.metadata-retain-period", 10*time.Minute, "Period at which metadata we have not seen will remain in memory before being deleted.")
 
 	f.DurationVar(&cfg.RateUpdatePeriod, "ingester.rate-update-period", 15*time.Second, "Period with which to update the per-user ingestion rates.")
+	f.DurationVar(&cfg.ActiveSeriesUpdatePeriod, "ingester.active-series-update-period", 5*time.Minute, "How often to update active series metrics (blocks engine only).")
+	f.DurationVar(&cfg.ActiveSeriesIdle, "ingester.active-series-idle", 15*time.Minute, "After what time is series considered to be inactive (blocks engine only).")
```

Comment:
Maybe we should call the CLI flag / YAML option `active-series-idle-time` or `active-series-idle-timeout`?